app/packages/state/src/recoil/schemaSettings.atoms.ts

```diff
@@ -309,3 +309,16 @@ export const selectedFieldsStageState = atom<any>({
     },
   ],
 });
+
+export const isFieldVisibilityActive = selector({
+  key: "isClearFieldVisibilityVisible",
+  get: ({ get }) => {
+    const isSelectedFieldsStageActive = get(fos.selectedFieldsStageState);
+    const affectedCount = get(fos.affectedPathCountState);
+
+    return isSelectedFieldsStageActive && affectedCount > 0;
+  },
+  cachePolicy_UNSTABLE: {
+    eviction: "most-recent",
+  },
```

Comment:
I'm not sure why I started adding these. Feel free to remove, I don't think it has performance value